### PR TITLE
feat(count-words): count occurrences of given string

### PR DESCRIPTION
**Pull request by isabelroses**
Like so
image

P.S. rust analyzer kept screaming at me about the missing semi-colons

### CODE REVIEW

**Review by uncenter — APPROVED** [review]
As discussed I've updated the count_words impl so that it returns a list of the words. :)

**Comment by uncenter** [comment]
Oops wrong button lmao.

**Comment by isabelroses** [comment]
:p formating

File: src/main.rs [hunk 1]
```diff
@@ -103,45 +104,56 @@ fn main() {
...
+						count_words::count_words(text.to_string())
+							.words
+							.iter()
+							.filter(|&w| *w == word)
+							.count()
+					);
+				} else {
+					println!("{}", count_words::count_words(text.to_string()).words.len());
+				}
 			}
```
Comment (uncenter) [inline src/main.rs]:
```suggestion
			Commands::CountWords { text, word } => {
				let words = count_words::count_words(text.to_string()).words;
				if let Some(word) = word {
					println!(
						"{}",
						words
							.iter()
							.filter(|&w| *w == word)
							.count()
					);
				} else {
					println!("{}", words.len());
				}
			}
```

**Review by uncenter — APPROVED** [review]

**Pull request merged**